tests/test_expertise_service.py

```diff
@@ -0,0 +1,451 @@
+
+        assert response[0]['name'] == 'test_run'
+        assert response[0]['status'].strip() == 'Error'
+        assert response[0]['description'] == 'use_title and use_abstract cannot both be False'
+        ###assert os.path.isfile(f"{server_config['WORKING_DIR']}/{job_id}/err.log")
+
+        # Clean up error job
+        shutil.rmtree(f"./tests/jobs/{openreview_context['job_id']}")
+    
+    def test_high_load(self, openreview_context, celery_session_app, celery_session_worker):
+        # Submit a working job and return the job ID
+        test_client = openreview_context['test_client']
+        num_requests = 5
+        id_list = []
+        # Make n requests
+        for _ in range(num_requests):
+            response = test_client.post(
+                '/expertise',
+                data = json.dumps({
+                        'name': 'test_run',
+                        'match_group': ["ABC.cc"],
+                        'paper_invitation': 'ABC.cc/-/Submission',
+                        "model": "elmo",
+                        "model_params": {
+                            "use_title": False,
+                            "use_abstract": True,
+                            "average_score": True,
+                            "max_score": False
+                        }
+                    }
+                ),
+                content_type='application/json'
+            )
+            assert response.status_code == 200, f'{response.json}'
+            job_id = response.json['job_id']
+            id_list.append(job_id)
+            response = test_client.get('/expertise/status', query_string={'id': f'{job_id}'}).json['results']
+            assert len(response) == 1
+            assert response[0]['name'] == 'test_run'
+            assert response[0]['status'] != 'Error'
+
+        # Check that the last request is queued
+        last_job_id = id_list[num_requests - 1]
+        response = test_client.get('/expertise/status', query_string={'id': f'{job_id}'}).json['results']
+        assert response[0]['status'] == 'Queued'
+        assert response[0]['name'] == 'test_run'
+        assert response[0]['description'] == 'Job is waiting to start fetching OpenReview data'
+        assert id_list is not None
+        openreview_context['job_id'] = id_list
+    
+    def test_fetch_high_load_results(self, openreview_context, celery_session_app, celery_session_worker):
+        assert openreview_context['job_id'] is not None
+        id_list = openreview_context['job_id']
+        num_requests = len(id_list)
+        test_client = openreview_context['test_client']
+        last_job_id = id_list[num_requests - 1]
+
+        # Assert that the last request completes
+        response = test_client.get('/expertise/status', query_string={'id': f'{last_job_id}'}).json['results']
+        assert len(response) == 1
+        while response[0]['status'] != 'Completed':
```

Comment:
This can be an infinite loop. Fix the number of tries and mark the test as failure after that.